docs/plans/plugins/HOST-DATA-API.proto

```
@@ -0,0 +1,218 @@
+// layer (which publishes task.* events), never the repository.
+
+syntax = "proto3";
+package kandev.plugin.v1;
+
+import "google/protobuf/struct.proto";
+
+// ── Added to the existing `service Host` (state/secrets/emit already there) ──
+service HostData {
+  // Reads — capability api_read:<resource>
+  rpc ListTasks(ListTasksRequest) returns (ListTasksResponse);
+  rpc GetTask(GetTaskRequest) returns (Task);
```

Buf (1.71.0)

[error] "kandev.plugin.v1.Task" is used as the request or response type for multiple RPCs.

(RPC_REQUEST_RESPONSE_UNIQUE)